actionpack/lib/action_dispatch/system_testing/test_helpers/screenshot_helper.rb

```diff
@@ -38,14 +37,32 @@ def save_image
             page.save_screenshot(Rails.root.join(image_path))
           end
 
+          def output_type
+            # Environment variables have priority
+            output_type = ENV["RAILS_SYSTEM_TESTING_SCREENSHOT"] || ENV["CAPYBARA_INLINE_SCREENSHOT"]
```

Comment:
This PR is looking good. I think however this does need to be documented in the take_screenshot method so folks know how to use these environment variables.